### PR TITLE
Ft send receive notification

### PR DESCRIPTION
**Pull request by habaumugisha1**
# Description

This PR contains is about sending and receiving in-app notifications and email notification when travel request approved, rejected, and conceled

## Type of change

Please check the options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How should this be manually tested or reproduced?

- Clone this repo
- Checkout to `ch-setup-testing-env`
- Run the command `npm i`
- Run the command `npm run test` to run tests locally 
- If you push new changes, the test shall run on travis

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

# Screenshots (if applicable)
Screen Shot 2020-11-24 at 09 24 49

Screen Shot 2020-11-24 at 09 25 15

Screen Shot 2020-11-24 at 09 25 33

### CODE REVIEW

File: src/config/pusher.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import Pusher from 'pusher';
+
+const pusher = new Pusher({
+  appId: '1110980',
```
Comment (salviosage) [inline src/config/pusher.js]: can you put that in .emv file

**Review by salviosage — COMMENTED** [review]
I left some comments , also please look into all aspects one need to be notofied and add notification functionality , aslo attach screenshoots in your PR descriptions

File: src/config/pusher.js (hunk 1 above)
Comment (habaumugisha1) [reply src/config/pusher.js]: Yeah sure

**Review by salviosage — COMMENTED** [review]
Please rebase

**Review by salviosage — APPROVED** [review]
LGTM

**Pull request merged**